catalog/spatial/geocoding/spatial-geocoding-admin-module/src/main/java/org/codice/ddf/spatial/admin/module/service/Geocoding.java

```diff
@@ -32,7 +32,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(Geocoding.class);
 
-  private static final String BASE_DIR = System.getProperty("user.dir") + "/content/store/";
+  private static final String BASE_DIR = System.getProperty("ddf.home") + "/content/store/";
```

Comment:
❓ The content store's directory is located under `${ddf.home}/data/content/store`. Should we use the same directory? That way we wouldn't have to change the `default.policy` file.

Reply:
Also, should be building up a Path instead of concatenating strings.